test/account-manager.js

```diff
@@ -0,0 +1,362 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs-extra')
+const chai = require('chai')
+const expect = chai.expect
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+chai.use(sinonChai)
+chai.should()
+
+const rdf = require('rdflib')
+const LDP = require('../lib/ldp')
+const SolidHost = require('../lib/models/solid-host')
+const AccountManager = require('../lib/models/account-manager')
+const UserAccount = require('../lib/models/user-account')
+const WebIdTlsCertificate = require('../lib/models/webid-tls-certificate')
+
+const testAccountsDir = path.join(__dirname, 'resources', 'accounts')
+
+var host
+
+beforeEach(() => {
+  host = SolidHost.from({ serverUri: 'https://example.com' })
+})
+
+describe('AccountManager', () => {
```

Comment:
Out of scope for this PR; our whole email subsystem needs testing and refactoring.